components/gitpod-cli/cmd/credential-helper.go

```diff
@@ -148,7 +148,7 @@ func parseHostFromStdin() string {
 
 func parseProcessTree() (repoUrl string, gitCommand string) {
 	gitCommandRegExp := regexp.MustCompile(`git(,\d+\s+|\s+)(push|clone|fetch|pull|diff)`)
-	repoUrlRegExp := regexp.MustCompile(`\sorigin\s*(https:.*\.git)\s`)
+	repoUrlRegExp := regexp.MustCompile(`\sorigin\s*(https:[^s]*)\s`)
```

Comment:
Did you mean `[^\s]` (not whitespace) instead of `[^s]` not `s`?

```suggestion
	repoUrlRegExp := regexp.MustCompile(`\sorigin\s*(https:[^\s]*)\s`)
```

Reply:
Good catch.